src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,20 @@
+package duke;
+
+public class DukeException extends Exception {
+    private final String message;
+
+    /**
+     * Creates an exception using a message.
+     *
+     * @param message The message to return in toString method.
+     */
+    public DukeException(String message) {
+        super(message);
+        this.message = message;
```

Comment:
Hi, I suppose you could have done without `super(message);` and it will still work as intended? 
Let me know if there is actually an intended effect of your `super(message);` 😃

Reply:
Yeah its redundant, thanks for noticing. Will remove.